pkg/cli/image/convert/convert.go

```diff
@@ -0,0 +1,129 @@
+package convert
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	apicfgv1 "github.com/openshift/api/config/v1"
+	operatorv1alpha1 "github.com/openshift/api/operator/v1alpha1"
+
+	"github.com/openshift/oc/pkg/cli/image/strategy"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/templates"
+	"sigs.k8s.io/yaml"
+)
+
+type ConvertOptions struct {
+	genericclioptions.IOStreams
+
+	ICSPFiles []string
+}
+
+func NewConvert(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
+	o := NewConvertOptions(streams)
+	cmd := &cobra.Command{
+		Use:   "convert ICSP-FILE",
+		Short: "Convert imagecontentsourcepolicy file(s) to imagedigestmirrorset file(s).",
+		Long: templates.LongDesc(`
+		Convert imagecontentsourcepolicy file(s) to imagedigestmirrorset file(s). The imagedigestmirrorset file(s) that can be added to a cluster will be written to file(s) under the temporary directory.
```

Comment:
The files should be written to either current directory or allow users to specify it with `--dest-dir`

Reply:
Or as an optional second parameter.